dll/win32/shell32/brfolder.cpp

```diff
@@ -408,8 +408,7 @@ BrFolder_Expand(
     ULONG ulFetched;
     while (S_OK == pEnum->Next(1, &pidlTemp, &ulFetched))
     {
-        if (!BrFolder_InsertItem(info, lpsf, pidlTemp, pidlFull, hParent))
-            break;
+        BrFolder_InsertItem(info, lpsf, pidlTemp, pidlFull, hParent);
```

Comment:
Add a code comment about why "failure" is ignored?

Reply:
Ok, done